packages/ember/tests/routing/decoupled_basic_test.js

```diff
@@ -507,4 +507,37 @@ class extends ApplicationTestCase {
     });
   }
 
+
+  ['@test The Specials Page defaults to looking models up via `find`'](assert) {
+    this.router.map(function() {
+      this.route('home', { path: '/' });
+      this.route('special', { path: '/specials/:menu_item_id' });
+    });
+    //
+    // this.applicationInstance.MenuItem = EmberObject.extend();
```

Comment:
@mixonic / @rwjblue, what's the proper way to convert this sort of thing? 

Currently, I get the error:
`Error: Assertion Failed: You used the dynamic segment menu_item_id in your route special, but undefined.MenuItem did not exist and you did not override your route's `model` hook.`

not sure what undefined.MenuItem is / where it's coming from.

Reply:
@NullVoxPopuli thanks for taking this work on.

I would expect the commented section here would be written as:

```js
let MenuItem = EmberObject.extend();
MenuItem.reopenClass({
  find(id) {
    return MenuItem.create({id});
  }
});
this.add('model:menu-item', MenuItem);
```

Reply:
still getting 
```
Error: Assertion Failed: You used the dynamic segment menu_item_id in your route special, but undefined.MenuItem did not exist and you did not override your route's `model` hook.
``` 

:-\